databuilder/query_model.py

```diff
@@ -13,6 +15,8 @@
 
 # The below classes and functions are the public API surface of the query model
 __all__ = [
+    "Node",
+    "Value",
```

Comment:
@evansd I think that `Value` is an oversight, but maybe you deliberately left off `Node` since it shouldn't be needed for defining datasets. I found that it was in use (for example in the query engine) which resulted in warnings from my IDE. I can be talked out of including it here.